File: atcoder/abc050A_addition_and_substraction_easy.py
# Vicfred
# https://atcoder.jp/contests/abc050/tasks/abc050_a
# simulation
A, op, B = input().split()

A = int(A)
B = int(B)

answer = 0
if op == '+':
    answer = A+B
else:
    answer = A-B

print(answer)

File: atcoder/abc050A_addition_and_substraction_easy.nim
# uninhm
# https://atcoder.jp/contests/abc050/tasks/abc050_a
# implementation

import strutils

let s = stdin.readLine.split

if s[1] == "+":
  echo s[0].parseInt + s[2].parseInt
else:
  echo s[0].parseInt - s[2].parseInt
